### PR TITLE
1 frontend code refactoring

### LINKED ISSUE #1: [frontend] Code refactoring
1. Find out how the app works
2. Try to run the application, you can use the script https://github.com/Kv-126-DevOps/None/issues/15#issuecomment-1079721724
3. In your opinion, add the necessary changes to the code
    - Format code according to PEP-8
    - Add check of required environment variables; set other environment variables to default values
    - Rename the environment variables according to the accepted standard naming convention
    - Fill out README.md according to the [README_service_template.md](https://github.com/Kv-126-DevOps/None/blob/main/README_service_template.md)
4. Create a pull request from the branch `issue-<ISSUE NUMBER>`

### PR DESCRIPTION
**Pull request by redbearddog**
[frontend] Code refactoring

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,11 @@
+POSTGRES_HOST=some_postgres_host
```
Comment (anatolek) [inline .env.example]: Not sure that this file is needed, so I guess it is better just add this info to the README.md and then remove this file at all

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM python:3.9-slim
+
+RUN useradd serve
+
+WORKDIR /home/serve
+
+COPY requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+COPY app app
```
Comment (anatolek) [inline Dockerfile]: You can use `--chown` option with COPY if you wish, see https://docs.docker.com/engine/reference/builder/#copy

File: README.md [hunk 3]
```diff
@@ -1 +1,44 @@
...
\ No newline at end of file
+
+### Frontend app
+
+    git clone --branch 1_frontend_code_refactoring https://github.com/Kv-126-DevOps/frontend.git /opt/frontend
+    docker run --network=kv126 -d --name frontend -e RESTAPI_HOST=rest-api -e RESTAPI_PORT=5000 -v /opt/frontend:/app -p 80:5000 python:3.9-slim sleep infinity
+    docker exec frontend pip install -r /app/requirements.txt
+    docker exec -d frontend bash -c "cd /app && flask run --host=0.0.0.0"
+
+### Setuping and Starting microservice
```
Comment (anatolek) [inline README.md]:
```suggestion
### Setting up and running a microservice
```

File: app/routes.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+from flask import render_template, redirect, url_for, Flask
+from app import app
+import requests
+import json
+import os
+
+HOST = os.environ.get('RESTAPI_HOST')
+PORT = os.environ.get('RESTAPI_PORT')
```
Comment (anatolek) [inline app/routes.py]:
```suggestion
PORT = os.environ.get('RESTAPI_PORT', 5000)
```

**Review by anatolek — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (redbearddog) [reply .env.example]: Deleted

File: README.md (hunk 3 above)
Comment (redbearddog) [reply README.md]: OK

File: app/routes.py (hunk 4 above)
Comment (redbearddog) [reply app/routes.py]: +

**Comment by redbearddog** [comment]
port added
.env.examples deleted
readme.md changed

File: Dockerfile (hunk 2 above)
Comment (redbearddog) [reply Dockerfile]: fixed

**Comment by redbearddog** [comment]
--chown=kvuser:kvuser

**Review by anatolek — APPROVED** [review]

**Pull request merged**